### DISCUSSION TITLE
Classification Store does not show options

### DISCUSSION THREAD

**Discussion by designjpp** [pimcore/pimcore#15721] [Q&A / Support]
Hello, 

we are in the process of implementing Pimcore 10.6 and we have a problem. Classification Store options no longer appear 
![image](https://github.com/pimcore/pimcore/assets/60102400/63e7c01c-418b-4b14-b616-aa78ccc1d12f)

And we already have some options created. Here the options inside the class
![image](https://github.com/pimcore/pimcore/assets/60102400/dfd8ebab-6aca-4da1-b464-545e959dd59c)

We suspect that it may have been the accent in the name that could have caused the problem.

Can anyone help us to solve it?

**Comment by dvesh3** [comment]
Hi @designjpp please check the response from the server, if it throws an error?
![image](https://github.com/pimcore/pimcore/assets/5137917/f55f29b1-26fb-4430-8c6e-75e3fcee8a45)

**Reply by designjpp** [reply]
Hello @dvesh3, 

A detail that I forgot to say before, this problem is in Prod mode.
In order not to interrupt the implementation of the information, we found that if we change APP_DEBUG=1 in the env file, it works.

Let's return to 0, to see if there are any errors

**Reply by designjpp** [reply]
This are logs details

`
2023-08-10T16:26:23-0300 : 2|Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\ClassificationstoreController::storetreeAction|pimcore_admin_dataobject_classificationstore_storetree|[]|{"node":"root"}
`

`
[10-Aug-2023 16:26:23 America/Sao_Paulo] PHP Deprecated:  htmlspecialchars(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php on line 1515
`

`
Deprecated: htmlspecialchars(): Passing null to parameter #1 ($string) of type string is deprecated in /var/www/html/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php on line 1515
[{"id":1,"text":"Bicicleta","expandable":false,"leaf":true,"expanded":true,"description":"","iconCls":"pimcore_icon_classificationstore","qtitle":"ID: 1","qtip":" "},{"id":3,"text":"Lube","expandable":false,"leaf":true,"expanded":true,"description":"","iconCls":"pimcore_icon_classificationstore","qtitle":"ID: 3","qtip":" "},{"id":4,"text":"Suplementacao","expandable":false,"leaf":true,"expanded":true,"description":"","iconCls":"pimcore_icon_classificationstore","qtitle":"ID: 4","qtip":" "},{"id":2,"text":"Vestuario","expandable":false,"leaf":true,"expanded":true,"description":"","iconCls":"pimcore_icon_classificationstore","qtitle":"ID: 2","qtip":" "}]
`
